app/include/zmk/ble.h

```diff
@@ -19,6 +19,12 @@
 #define ZMK_BLE_PROFILE_COUNT CONFIG_BT_MAX_PAIRED
 #endif
 
+enum zmk_ble_advertising_type {
+    ZMK_ADV_NONE,
+    ZMK_ADV_DIR,
+    ZMK_ADV_CONN,
```

Comment:
Should really all have the full `ZMK_BLE_` prefix now that it's moved somewhere more public.